presto-docs/src/main/sphinx/release/release-0.146.rst

```diff
@@ -6,6 +6,7 @@ General Changes
 ---------------
 
 * Require at least 4096 file descriptors to run Presto.
+* Casting ``MAP`` types is supported.
```

Comment:
@martint gave a good way to think about this: If you prefix the release note with "After updating, Presto will ..."

Thus, compare these two:

* After updating, Presto will casting map types is supported.
* After updating, Presto will support casting between map types.